src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -6688,7 +6692,20 @@ def _update_pipeline_run_status(
                 ExecutionStatus.FAILED,
             }:
                 run_update.end_time = datetime.utcnow()
-
+                with track_handler(
+                    AnalyticsEvent.RUN_PIPELINE_ENDED
+                ) as analytics_handler:
+                    analytics_handler.metadata = {
+                        "pipeline_run_id": pipeline_run_id,
+                        "status": new_status,
+                        "num_steps": num_steps,
+                        "start_time": pipeline_run.start_time.strftime(
```

Comment:
just FYI, we don't send the `start_time` of the pipeline run in the other event, so there won't be any way to match that I assume. Not sure if we either want to add that in the other analytics event, or remove it here or maybe just send the duration here if that is interesting?